var/spack/repos/builtin/packages/openblas/openblas_fujitsu.patch

```diff
@@ -0,0 +1,14 @@
+--- OpenBLAS-0.3.6/f_check	2019-04-30 02:22:19.000000000 +0900
++++ OpenBLAS-0.3.6/f_check_fix	2019-08-05 10:31:56.066962193 +0900
+@@ -69,7 +69,10 @@
+ 	    $bu       = "_";
+ 	}
+ 
+-	if ($data =~ /GNU/) {
++        if ($data =~ /Fujitsu/) {
```

Comment:
I see that this script uses always `if` and never `elsif` in this section. Is there any reason why you changed this pattern for Fujitsu?

Reply:
I used `elsif` because `GNU` was included in `$data` result when building with Fujitsu compiler too.

Reply:
Thank you @alalazo !
Any comments on this? We are waiting for this pull request to be merged. 👍